support-frontend/conf/routes

```diff
@@ -170,6 +170,7 @@ GET /p/:promoCode                                                 controllers.Pr
 
 # ----- Prices API ----- #
 GET /prices                                                       controllers.PricesController.getPrices()
+GET /pricesWithSummary                                            controllers.PricesController.getPricesWithSummary()
```

Comment:
since it's almost the same it could even be `/prices?show-summary`

Reply:
That was my original plan, however upon trying I assumed it wasn't possible/advisable to read query parameters with a `CachedAction`, and I didn't want to change it to a `NoCacheAction`?

Reply:
Ah right. Perhaps we should change this to a regular `Action`? Currently [SDC polls](https://github.com/guardian/support-dotcom-components/blob/main/packages/server/src/productPrices.ts#L15) the endpoint every minute (for each instance) which isn't much. I guess it depends what it's going to be used for long term

Reply:
Ok I'll do that.